CITATION.cff

```diff
@@ -0,0 +1,57 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+
+# Project information
+authors:
+  - name: "National Optical Astronomy Observatories"
+  - name: "The IRAF community"
+title: IRAF
+type: software
```

Comment:
This is implicit; when there is no type, `software` is always assumed for the *main* data.  But a type for reference objects, including the `preferred-citation`, is mandatory.